config.py

```diff
@@ -0,0 +1,42 @@
+from __future__ import annotations
+
+from collections import UserDict
+from typing import Any
+
+import logging
+import sys
+
+import yaml
+
+
+class Config(UserDict):
+    """defines the structure of the config object to be passed to the function extract_UKBB_tabular_data"""
+
+    FieldIDs: list[int]
+    InstanceIDs: list[int] | list[None]
+    SubjectIDs: list[int] | list[None]
+    ArrayIDs: list[int] | list[None]
+    Categories: list[Any]
+    recode_field_names: bool
+    recode_data_values: bool
+    drop_empty_strings: bool
+    drop_extra_NA_codes: bool
+    wide: bool
+    recode_wide_column_valuetypes: bool
+    convert_compound_to_list: bool
+    convert_less_than_value_integer: Any
```

Comment:
This must be an integer.

Reply:
0da710d4cfd04ddcd5c058220abdd2b13ff40122